lib/fs-watch/Cargo.toml

```diff
@@ -1,7 +1,10 @@
 [package]
 name = "linkerd2-fs-watch"
 version = "0.1.0"
-authors = ["Oliver Gould <ver@buoyant.io>"]
+authors = [
+    "Eliza Weisman <eliza@buoyant.io>",
```

Comment:
Probably out of scope for this branch, but I think it it _might_ be a good idea to add people to the code owners file for some of the crates that are being factored out, so they're automatically added to PRs touching the code they're most familiar with?

Reply:
sounds reasonable but not in scope